main/AndroidManifest.xml

```diff
@@ -82,6 +82,9 @@
             android:name="com.google.android.geo.API_KEY"
             android:value="@string/maps_api2_key" />
 
+        <!-- required by Google Maps SDK version 16 and below if targeted to SDK 28 or above -->
+        <uses-library android:name="org.apache.http.legacy" android:required="false" />
```

Comment:
please write it in multi line form like other code
~~~
        <uses-library
            android:name="org.apache.http.legacy"
            android:required="false" />
~~~

Reply:
formatting updated accordlingly